package.json

```diff
@@ -28,25 +28,24 @@
   "devDependencies": {
     "@babel/core": "^7.18.13",
     "@babel/preset-env": "^7.18.10",
-    "@commitlint/cli": "^17.0.3",
+    "@commitlint/cli": "^17.1.2",
     "@lwc/eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
     "@lwc/jest-utils-lwc-internals": "link:./scripts/jest/utils",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
-    "@rollup/plugin-typescript": "^8.3.4",
-    "@types/acorn": "^4.0.6",
+    "@rollup/plugin-typescript": "^8.4.0",
     "@types/babel__core": "^7.1.19",
     "@types/glob": "^7.2.0",
-    "@types/jest": "^28.1.7",
-    "@types/node": "^18.7.11",
-    "@typescript-eslint/eslint-plugin": "^5.34.0",
-    "@typescript-eslint/parser": "^5.34.0",
+    "@types/jest": "^28.1.8",
+    "@types/node": "^18.7.13",
+    "@typescript-eslint/eslint-plugin": "^5.35.1",
+    "@typescript-eslint/parser": "^5.35.1",
     "babel-plugin-tester": "^10.1.0",
     "bundlesize": "^0.18.1",
     "es5-proxy-compat": "^0.22.4",
-    "eslint": "^8.22.0",
+    "eslint": "^8.23.0",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^26.8.7",
+    "eslint-plugin-jest": "^27.0.1",
```

Comment:
Biggest change [here](https://github.com/jest-community/eslint-plugin-jest/releases/tag/v27.0.0) is that Node 12 is no longer supported. This should be fine for us, since this is just a devDep.